spec/react_on_rails/generators/dev_tests_generator_spec.rb

```diff
@@ -25,16 +25,16 @@
 
     it "changes package.json to use local react-on-rails version of module" do
       assert_file("package.json") do |contents|
-        assert_match('"react-on-rails"', contents)
-        assert_match('"postinstall"', contents)
+        expect(contents).to match('"react-on-rails"')
```

Comment:
Ideally, such changes are in a separate PR.

Reply:
Sure. But it was needed to pass Rubocop.